Gems/FPSProfiler/Code/Source/Clients/FPSProfilerSystemComponent.cpp

```diff
@@ -201,13 +194,18 @@ namespace FPSProfiler
         return m_configuration.m_SaveFpsData || m_configuration.m_SaveCpuData || m_configuration.m_SaveGpuData;
     }
 
-    void FPSProfilerSystemComponent::ChangeSavePath(const AZStd::string& newSavePath)
+    void FPSProfilerSystemComponent::ChangeSavePath(const AZ::IO::Path& newSavePath)
     {
-        AZ_Printf("FPS Profiler", "Path changed.");
+        if (!IsPathValid(newSavePath))
+        {
+            return;
+        }
+
         m_configuration.m_OutputFilename = newSavePath;
+        AZ_Warning("FPS Profiler", false, "Path changed.");
```

Comment:
```suggestion
        AZ_Warning("FPS Profiler", !m_isProfiling, "Path changed during activated profiling.");
```
Warning about changed path should be triggered only when there is any danger associated with it. Also warning message should include information about the danger

Reply:
Fair point - updated code.